tests/safety/test_subaru.py

```diff
@@ -7,35 +7,46 @@
 from functools import partial
 
 
-MSG_SUBARU_Brake_Status     = 0x13c
-MSG_SUBARU_CruiseControl    = 0x240
-MSG_SUBARU_Throttle         = 0x40
-MSG_SUBARU_Steering_Torque  = 0x119
-MSG_SUBARU_Wheel_Speeds     = 0x13a
-MSG_SUBARU_ES_LKAS          = 0x122
-MSG_SUBARU_ES_LKAS_ANGLE    = 0x124
-MSG_SUBARU_ES_Brake         = 0x220
-MSG_SUBARU_ES_Distance      = 0x221
-MSG_SUBARU_ES_Status        = 0x222
-MSG_SUBARU_ES_DashStatus    = 0x321
-MSG_SUBARU_ES_LKAS_State    = 0x322
-MSG_SUBARU_ES_Infotainment  = 0x323
+MSG_SUBARU_Brake_Status      = 0x13c
+MSG_SUBARU_CruiseControl     = 0x240
```

Comment:
these names are super long, you can properly namespace here, so they don't need to be that long

Reply:
scoped in an enum